rotkehlchen/chain/ethereum/defi/price.py

```diff
@@ -112,12 +112,10 @@ def _handle_curvepool_price(
 def handle_underlying_price_yearn_vault(
         ethereum: 'EthereumInquirer',
         contract: EvmContract,
-        div_decimals: int,
         asset_price: Price,
 ) -> FVal:
-    div_decimals = 18
     price_per_full_share = contract.call(ethereum, 'getPricePerFullShare')
-    usd_value = FVal(asset_price * price_per_full_share) / 10 ** div_decimals
+    usd_value = FVal(asset_price * price_per_full_share) / 10 ** 18
```

Comment:
Can you use `DEFAULT_TOKEN_DECIMALS` here?

Reply:
I knew there was a constant and couldn't find it. Thanks